--- a/ui/v2.5/src/components/Scenes/SceneList.tsx
+++ b/ui/v2.5/src/components/Scenes/SceneList.tsx
@@ -436,8 +436,8 @@
         <EditScenesDialog
           selected={selectedItems}
           onClose={onCloseEditDelete}
-          filter={filter}
-          selectAllItemCount={totalCount}
+          filter={true}
+          selectAllItemCount={999}
         />
       );
     }, [showModal, selectedItems, onCloseEditDelete, filter, totalCount]);
